Common/src/Storage/TaskTableExtensions.cs

```diff
@@ -640,10 +608,6 @@ public static async Task<TaskData> ReleaseTask(this ITaskTable   taskTable,
                                             cancellationToken: cancellationToken)
                              .ConfigureAwait(false);
 
-    taskTable.Logger.LogInformation("Released task {task} on {podName}",
-                                    taskData.TaskId,
-                                    taskData.OwnerPodId);
-
     taskTable.Logger.LogDebug("Released task {taskData}",
                               res);
```

Comment:
I would have kept the first log, and pass it to Debug.

Reply:
It is redundant with the log from TaskTable.UpdateOneTask.

Reply:
In that case, you could also remove the `LogDebug` just after

Reply:
As it shows the content of the data, not just what is modified, we can keep it